src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -589,6 +580,16 @@ class IOpenGL_PhysicalDeviceBase : public IOpenGLPhysicalDeviceBase
 			GetIntegerv(GL_MAX_TESS_EVALUATION_INPUT_COMPONENTS, reinterpret_cast<GLint*>(&m_properties.limits.maxTessellationEvaluationInputComponents));
 			GetIntegerv(GL_MAX_TESS_EVALUATION_OUTPUT_COMPONENTS, reinterpret_cast<GLint*>(&m_properties.limits.maxTessellationEvaluationOutputComponents));
 		}
+		if (m_glfeatures.isFeatureAvailable(m_glfeatures.NBL_ARB_geometry_shader4))
```

Comment:
`ARB_geometry_Shader4` is a desktop GL extension, which we don't need to care about cause the entire feature is core in GL 4.4

Reply:
the check should be
```
if (!IsGLES || Version>=3.2.0)
{
no suffix
}
else if (OES_geometry_shader)
{
OES suffices
}
else if (EXT_geometry_shader)
{
EXT suffices
}
else if (INTEL_geometry_shader)
{
INTEL suffices
}
else
    m_features.geometryShader = false;
```